srcpkgs/clickclack/patches/support-cross-compilation.patch

```diff
@@ -0,0 +1,22 @@
+ Makefile | 3 ++-
+ 1 file changed, 2 insertions(+), 1 deletion(-)
+
+diff --git a/Makefile b/Makefile
+index b593b7f..1bf7a8e 100644
+--- a/Makefile
++++ b/Makefile
+@@ -2,12 +2,13 @@ PREFIX:=/usr
+ 
+ PROGRAMS = clickclack
+ 
++CC ?= $(CROSS_COMPILE)gcc
```

Comment:
$CC is already set by make and we override it, so I don't think we have to patch this